www/docs/en/dev/guide/hybrid/plugins/index.md

```diff
@@ -44,13 +44,12 @@ For each corresponding native interface, see the list at the end of
 this section.
 
 In addition to these instructions, when preparing to write a plugin it
-is best to look over
-[existing plugins](http://cordova.apache.org/contribute)
+is best to look over [existing plugins](http://cordova.apache.org/contribute)
 for guidance.
 
 ## Building a Plugin
 
-Application developers use the CLI's [plugin add command](../../../cordova-cli/index.html#cordova-plugin-add-command) to add a plugin to a project. The
```

Comment:
@nikhilkh this heading doesn't exist in the master cordova-cli reference. I assumed you meant `cordova-plugin-command`. Is that correct?